File: src/javascript/user/contact/get-contact.mjs
import { FaxCoreClient, printResult } from '../../common/faxcoreClient.mjs';

const contactID = process.env.FAXCORE_CONTACT_ID;
if (!contactID) {
  throw new Error('FAXCORE_CONTACT_ID is required.');
}

const client = new FaxCoreClient();
const result = await client.get('/api/contact', {
  'model.contactID': Number(contactID)
});

printResult('Get contact', result);

File: src/dotnet/FaxCore.ApiSamples/Samples/User/Contact/GetContact.cs
using System;
using System.Threading.Tasks;

namespace FaxCore.ApiSamples.Samples.User.Contact
{
    internal static class GetContact
    {
        public static Task<string> RunAsync(FaxCoreClient client, SampleConfig config)
        {
            if (!config.ContactID.HasValue)
            {
                throw new InvalidOperationException("Set FAXCORE_CONTACT_ID before running user.contact.get.");
            }

            return client.GetAsync("/api/contact?model.contactID=" + Uri.EscapeDataString(config.ContactID.Value.ToString()));
        }
    }
}
